src/OnePlusBot/Modules/Administration.cs

```diff
@@ -62,19 +88,19 @@ public async Task<RuntimeResult> BanAsync(IGuildUser user, [Remainder] string re
             {
                 try 
                 {
-                    const string banMessage = "You were banned on r/OnePlus for the following reason: {0}\n" +
+                    const string banDMMessage = "You were banned on r/OnePlus for the following reason: {0}\n" +
```

Comment:
In this case, rather than having a double capital M, I'd make use of comments to clarify it's use case. 

Generally commenting everything is something planned, too.

Reply:
If the variable can explain its purpose within the name, I prefer that to be honest